Source/Analytics/Web/src/components/Reports/ReportsPerHealthRiskPerDay.js

```diff
@@ -0,0 +1,154 @@
+            reportsPerHealthRisk: [],
+            regions: [],
+            healthRisks: [''],
+            selectedHealthRisk: '',
+            isLoading: true,
+            isError: false
+        };
+    }
+
+    fetchAllRegions() {
+        const allRegionsQuery = new AllRegions();
+        this.queryCoordinator.execute(allRegionsQuery).then(result => {
+            if (result.success) {
+                this.setState({
+                    regions: result.items
+                });
+            }
+        }).catch(_ => {
+            this.setState({
+                isLoading: false,
+                isError: true
+            });
+        });
+    }
+
+    fetchReportsPerHealthRiskLastWeek() {
+        const reportsPerHealthRisk = new CaseReportsPerHealthRiskPerDayQuery();
+        reportsPerHealthRisk.numberOfDays = 7;
+
+        this.queryCoordinator.execute(reportsPerHealthRisk).then(queryResult => {
+            if (queryResult.success && queryResult.items.length > 0) {
+                const json = queryResult.items;
+                var healthRisks = this.getAllHealthRisksFromReports(json);
+                this.setState({
+                    reportsPerHealthRisk: json,
+                    healthRisks: healthRisks,
+                    selectedHealthRisk: healthRisks[0],
+                    isLoading: false,
+                    isError: false
+                });
+            }
+        }).catch(_ => {
+            this.setState({
+                isLoading: false,
+                isError: true
+            });
+        }
+        );
+    }
+
+    getAllHealthRisksFromReports(reports) {
+        const healthRisks = [];
+        reports.forEach(report => {
+            var risks = Object.keys(report.reportsPerHealthRisk);
+            risks.forEach(risk => {
+                if (healthRisks.indexOf(risk) < 0) {
+                    healthRisks.push(risk);
+                }
+            })
+        });
+        return healthRisks;
+    }
+
+    componentDidMount() {
+        this.queryCoordinator = new QueryCoordinator();
+        this.fetchAllRegions();
+        this.fetchReportsPerHealthRiskLastWeek();
+    }
+
+    saveSelectedValue(event) {
+        this.setState({
+            selectedHealthRisk: event.target.value
+        });
+    }
+
+    renderReports(region) {
+        if (this.state.reportsPerHealthRisk.length > 0) {
+            return this.state.reportsPerHealthRisk.map((report, i) => {
+                const reports = report.reportsPerHealthRisk[this.state.selectedHealthRisk];
+                const reportsInRegion = reports[region];
+                if (reportsInRegion != null)
+                    return <TableCell className="cell" key={i}>{reportsInRegion}</TableCell>;
+                else
+                    return <TableCell className="cell" key={i}>-</TableCell>;
+            });
+        }
+    }
+
+    renderRegions() {
+        return this.state.regions.map(region => {
+            return (
```

Comment:
I think you can skip the second `return` statement, by mapping like this: `...region.map(region => (<TableRow>...</TableRow>)`